### ISSUE TITLE
Document which node version is needed in order to run this project

### ISSUE THREAD

**Issue by HarelM** [Turfjs/turf-www#182]
Hi,

I've tried to make this work locally and these are the impressions I got:
1. It's not clear which node version to use, I used 16 and 18 and both failed, I think 14 is working.
2. It's worth adding that as a .nvmrc file, write it in the contributing guide or both.
3. I couldn't see any github actions that run as part of this repo, these help in both keeping the code working and describing how to create a working environment.
4. There are two part of the initial setup that are not clear if both are needed or not: `npm start` and after that there's the `npm generate` and `npx ...`. Will these two options get the same results?

Thanks for all the hard work put into this! If there's something I can do to help please let me know.

**Comment by JamesLMilner** [comment]
Hi @HarelM thanks for raising. Yes it appears that the site doesn't appear to be working with Node 18 at the moment (I managed to get this working with Node 14 and 16 okay). Many of the dependencies are very out of date and would need upgrading in order to get this working I think - I did try to get it working for 18 without doing this but it became complex quickly.

I think using an .nvmrc file and stating the current require version to be Node 14 until we can figure out how to get it working across different versions would be a good start. I think it would make sense to use github actions to ensure builds work across different versions (we could use a build matrices).

If you have a good idea on how to resolve these issues PRs are very welcome!
